setup.py

```diff
@@ -4,40 +4,14 @@
 from shutil import copyfile, rmtree
 import os
 import glob
-
-def build_package():
-    try:
-        rmtree('pyaf')
-    except:
-        pass
-    os.mkdir('pyaf')
-    os.mkdir('pyaf/TS')
-    for file in glob.glob('TS/*.py'):        
-        copyfile(file, 'pyaf/' + file)
-    copyfile('ForecastEngine.py' ,
-             'pyaf/ForecastEngine.py')
-    copyfile('HierarchicalForecastEngine.py',
-             'pyaf/HierarchicalForecastEngine.py')
-    copyfile('__init__.py',
-             'pyaf/__init__.py')
-    os.mkdir('pyaf/CodeGen')
-    for file in glob.glob('CodeGen/*.py'):        
-        copyfile(file, 'pyaf/' + file)
-    os.mkdir('pyaf/Bench')
-    for file in glob.glob('Bench/*.py'):        
-        copyfile(file, 'pyaf/' + file)
-    
-
-build_package();
-
 setup(name='pyaf',
       version='1.0',
       description='Python Automatic Forecasting',
       author='Antoine CARME',
       author_email='antoine.carme@laposte.net',
       url='https://github.com/antoinecarme/pyaf',
       license='BSD 3-clause',
-      packages=['pyaf' , 'pyaf.TS' ,  'pyaf.CodeGen' ,  'pyaf.Bench'],
+      packages=['pyaf'],
```

Comment:
@antoinecarme Please take a look at the proposed changes to setup.py right here. The significant change is that we are now only identifying a single "package" and adding __init__.py files to the package modules (TS, CodeGen, and Bench).

Reply:
There is no longer any need to copy files into the sub package directories at setup time.  I think this a more standard approach for python packaging